s3-uploader/runtimes/cpp11/Dockerfile

```diff
@@ -1,26 +1,36 @@
-FROM public.ecr.aws/lambda/provided:al2 as builder
+ARG ARCH
+FROM public.ecr.aws/lambda/provided:al2-$ARCH AS builder
```

Comment:
The `provided:al2` is already multi-arch so you can safely remove the `-$ARCH`
Actually the image won't be found otherwise

Reply:
you can also remove `ARG ARCH`